src/main/java/de/leonhard/storage/Config.java

```diff
@@ -56,6 +58,34 @@ public Config(final File file) {
     super(file);
   }
 
+  public void annotateClass(Object classInstance) {
+    Class<?> clazz = classInstance.getClass();
+    try {
+      for (Field field : clazz.getFields()) {
+        if(field.isAnnotationPresent(ConfigPath.class)) {
```

Comment:
Isn't this really slow, first checking If the annotation is present and then getting it?

I myself would use:
```java
ConfigPath configPath = field.getAnnotation(ConfigPath.class);
if (configPath != null) {
    field.setAccessible(true);
    field.set(classInstance, this.get(configPath.value(), field.getType()));
}
```